packages/sit-onyx/src/components/OnyxNavBar/modules/OnyxFlyoutMenu/OnyxFlyoutMenu.vue

```diff
@@ -99,17 +99,36 @@ const {
       font-family: var(--onyx-font-family);
       z-index: var(--onyx-z-index-flyout);
 
-      &--with-header {
-        padding-top: 0;
+      &-header {
+        position: sticky;
+        top: 0;
+
+        &:empty {
+          padding-top: var(--onyx-spacing-2xs);
+        }
       }
 
-      &--with-footer {
-        padding-bottom: 0;
+      &-footer {
+        position: sticky;
+        bottom: 0;
+
+        &:empty {
+          padding-bottom: var(--onyx-spacing-2xs);
+        }
       }
     }
 
     &__wrapper {
       padding: 0;
+      /**
+       * The last option should only be half visible:
+       * 7.5 * OnyxListItem, where OnyxListItem => 2 * padding + line-height of OnyxListItem 
+       */
+      max-height: calc(
+        (var(--onyx-flyout-menu-visible-item-count, 7) + 0.5) *
+          (2 * var(--onyx-density-xs) + 1.5rem)
```

Comment:
```suggestion
          (2 * var(--onyx-density-xs) + 1lh)
```

Can't we use the `lh` unit here for the line height?